nexus/reconfigurator/planning/src/blueprint_builder.rs

```diff
@@ -472,6 +475,44 @@ impl<'a> BlueprintBuilder<'a> {
         Ok(Ensure::Added)
     }
 
+    pub fn sled_ensure_zones_not_using_expunged_disks(
```

Comment:
This method is currently acting as a no-op, because of the TODOs below.

But I think the right action here would be to add the zones to an "expunged zone" field within the blueprint?

@sunshowers , do you think #5211 will support that? Or will I need to add a `BTreeSet` to the `BlueprintBuilder` for each zone type that could be removed?